client/service-worker.js

```diff
@@ -2,6 +2,62 @@
 /* global clients */
 "use strict";
 
+const excludedPathsFromCache = /^(?:socket\.io|storage|uploads|cdn-cgi)\//;
+
+self.addEventListener("install", function() {
+	self.skipWaiting();
```

Comment:
Where does this come from?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/skipWaiting